supervisor/plugins/manager.py

```diff
@@ -27,6 +28,11 @@ def __init__(self, coresys: CoreSys):
         self._observer: PluginObserver = PluginObserver(coresys)
         self._multicast: PluginMulticast = PluginMulticast(coresys)
 
+    @property
+    def all_plugins(self) -> list[PluginBase]:
+        """Return cli handler."""
+        return [self._cli, self._dns, self._audio, self._observer, self._multicast]
```

Comment:
Do we want to use Tuple aka `()`? Would be faster to generate and use less memory

Reply:
For 5 elements I don't really think it matters, but if you want I can change